index.html

```diff
@@ -8,27 +8,232 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="main">
-   <div class="container">
-       <header>
-           <img src="images/1.png">
-       </header>
-       <div class="navbar"> <img src="images/3.png"> </div>
-   </div>
-
-   <div class="content"> 
-
-    <div class="heading" ><h1>Charlie Giorgania</h1></div>
-    <div class="para">
-    <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+ <div class="container">
+<header>
+    <nav>
+    <a href="https://www.microverse.org/"> <img src="images/1.png" alt="Logo"> </a> 
+    <div class="navbar"> <img src="images/3.png" alt="Menu_icon"> </div>
+    </nav>
+</header>
+
+<section>
+    <div class="container content">
+        <h1>Charlie <br> Giorgania</h1>
+        <div class="paragraph">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </div>
```

Comment:
[Optional]
There's no need to wrap `<p>` tag with `<div>` tag because both are block-level elements. Kindly remove the `<div>` tag and apply the paragraph style on `<p>` tag.